Shadowlands/TazaveshTheVeiledMarket/Soleah.lua

```diff
@@ -30,15 +34,19 @@ function mod:OnBossEnable()
 
 	self:Log("SPELL_CAST_SUCCESS", "SummonAssassins", 351124)
 	self:Log("SPELL_CAST_START", "CollapsingStar", 353635)
+	self:Log("SPELL_CAST_SUCCESS", "CollapsingStarSummoned", 353635)
 	self:Log("SPELL_AURA_APPLIED", "CollapsingEnergyApplied", 350804)
 	self:Log("SPELL_AURA_APPLIED_DOSE", "CollapsingEnergyApplied", 350804)
 	self:Log("SPELL_AURA_REMOVED", "CollapsingEnergyRemoved", 350804)
 	self:Log("SPELL_CAST_START", "HyperlightSpark", 350796)
 	self:Log("SPELL_CAST_SUCCESS", "PowerOverwhelming", 351086)
 	self:Log("SPELL_AURA_REMOVED", "PowerOverwhelmingRemoved", 351086)
 	self:Log("SPELL_CAST_SUCCESS", "EnergyFragmentation", 351096)
+	self:Log("SPELL_CAST_START", "ShurikenBlitz", 351119)
 end
 
+local starCount = 4
```

Comment:
If you don't want to reset this on engage then it should probably just be scoped within the 2 functions that use it, since they are right next to each other and directly related. Otherwise it should be in the dedicated "locals" section like other modules.